lighthouse-core/audits/viewport.js

```diff
@@ -27,6 +27,7 @@ class Viewport extends Audit {
       category: 'Mobile Friendly',
       name: 'viewport',
       description: 'HTML has a viewport <meta>',
+      helpText: 'If your app doesn\'t have a viewport meta tag, it probably isn\'t optimized for mobile screens. <a href="https://developers.google.com/web/tools/lighthouse/audits/has-viewport-meta-tag" target="_blank" rel="noopener noreferrer">Learn more</a>.',
```

Comment:
should this be more like the DBW style `helpText`, like "Consider including a viewport meta tag, to optimize your app for mobile screens" or something?